docs/_docs/reference/other-new-features/trait-parameters.md

```diff
@@ -53,13 +53,19 @@ The correct way to write `E` is to extend both `Greeting` and
 class E extends Greeting("Bob"), FormalGreeting
 ```
 
-## Traits With Context Parameters
+## Default values and implicit parameters
 
-This "explicit extension required" rule is relaxed if the missing trait contains only
-[context parameters](../contextual/using-clauses.md). In that case the trait reference is
-implicitly inserted as an additional parent with inferred arguments. For instance,
-here's a variant of greetings where the addressee is a context parameter of type
-`ImpliedName`:
+If the trait `T` is parametrised such that `new T{}` would be a legal annonymous class creation, the "explicit extension required" rule does not apply, the parameters filled in as for the annonymous class:
+* Parameters with default values are set to their default value.
+* [Context parameters](../contextual/using-clauses.md) are set to the value present in the implicit scope. Notably from class parameters or other traits.
+
+```scala
+trait withLastName(val lastName: String = "")
+
+case class Person extends withLastName
```

Comment:
this makes a compile error:
```scala
scala> case class Person extends withLastName
-- [E004] Syntax Error: --------------------------------------------------------
1 |case class Person extends withLastName
  |           ^^^^^^
  |           A case class must have at least one parameter list
  |
  | longer explanation available when compiling with `-explain`
1 error found
```

Reply:
```suggestion
case class Person() extends withLastName
```
Refactoring mistake, the above solves the problem ^^